source/includes/_algorithms.md

```diff
@@ -0,0 +1,567 @@
+    },
+    "name": "my_first_algorithm",
+    "settings": {
+      "environment": "cpu",
+      "language": "java",
+      "license": "apl",
+      "network_access": "full",
+      "pipeline_enabled": true,
+      "source_visibility": "closed" 
+    }
+  }'
+```
+
+```python
+import Algorithmia
+
+client = Algorithmia.client('API_KEY')
+
+algo = client.algo(':username/my_first_algorithm')
+
+algo.create(
+  details = {
+    "label": "My First Algorithm"
+  },
+  settings = {
+    "language": "java",
+    "source_visibility": "closed",
+    "license": "apl",
+    "network_access": "full",
+    "pipeline_enabled": True,
+    "environment": "cpu"
+  }
+)
+```
+
+`POST /algorithms/:username`
+
+### Path Parameters
+
+|Parameter|Type|Description|
+|-|-|-|
+|username|String|*Required*. The username of the user or organization that will own this algorithm.|
+
+### Payload Parameters
+
+|Parameter|Type|Description|
+|-|-|-|
+|`details.label`|String|*Required*. The human-readable name for your algorithm.|
+|`details.summary`|String|A full description of your algorithm's capabilitities. HTML is accepted for rendering in the algorithm's "Docs" tab.|
+|`details.tagline`|String|A short description of your algorithm for display under the algorithm's label.|
+|`name`|String|*Required*. The unique ID for your algorithm. Only alphanumeric characters and underscores are permitted. Must begin with a letter.|
+|`settings.environment`|String|*Required if `language` specified*. The hardware your algorithm will run on. Choose from `cpu` or `gpu`.|
+|`settings.language`|String|*Required if `package_set` not specified*. The language you wish to develop your algorithm in. Choose from `java`, `javascript`, `python2-lackpack` (Python 2), `python3-1` (Python 3), `r`, `ruby`, `rust`, or `scala`. |
+|`settings.license`|String|*Required*. The license for your algorithm's source code. Choose from [`apl`](https://algorithmia.com/api_dev_terms), [`apache2`](https://www.apache.org/licenses/LICENSE-2.0), [`gpl3`](https://www.gnu.org/licenses/gpl-3.0.en.html) or [`mit`](https://opensource.org/licenses/MIT). Select `apl` if you wish for your algorithm's source code to be closed-source.|
+|`settings.network_access`|String|*Required*. Specifies whether the algorithm will have access to the public internet upon execution. Choose from `full` or `isolated`.|
+|`settings.package_set`|String|*Required if `language` not specified*. Specifies the package set that should define the algorithm's build and runtime environments.|
+|`settings.pipeline_enabled`|Boolean|*Required*. Specifies whether this algorithm is allowed to call other algorithms on the platform.|
+|`settings.source_visbility`|String|*Required*. Specifies whether the source code for this algorithm will be viewable by any user of the platform. If creating an algorithm with an external repository host, such as GitHub, determines whether the repo is created as public or private. Choose from `open` or `closed`.|
```

Comment:
:bulb: Update phrasing of "whether the repo is created as public or private" to "whether the created repository is public or private."